tests/test_events.py

```diff
@@ -0,0 +1,7 @@
+from fastapi.testclient import TestClient
+
+
+def test_post_events(client: TestClient):
```

Comment:
What is the reason behind strict typing this test fixture as a `TestClient`?  It looks like it works similarly without it - just noting because we weren't doing this in mbtools.?

Reply:
Yeah, it's just a type hint. Probably I used it because it was used in the FastAPI examples. In general these type hints aren't required which is why it works either way.